autoendpoint/src/extractors/subscription.rs

```diff
@@ -84,7 +84,9 @@ impl FromRequest for Subscription {
                 .fernet
                 .decrypt(&repad_base64(&token_info.token))
                 .map_err(|e| {
-                    error!("🔐 fernet: {:?}", e);
+                    // Since we're decrypting and endpoint, we get a lot of spam links.
+                    // This can fill our logs.
+                    trace!("🔐 fernet: {:?}", e);
```

Comment:
Question: doesn't `trace!` capture more detailed logging, traceback info? I'm just going off the comment in that the intention seems to reduce what is filled in the log, or is that not the intention. I may likely be missing something!

Reply:
Cool, thanks!